crates/daphne-service-utils/src/durable_requests/bindings/aggregation_job_store.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2025 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+use crate::{
+    aggregation_job_store_capnp::new_job_request,
+    capnproto::{CapnprotoPayloadDecode, CapnprotoPayloadEncode},
+    durable_requests::ObjectIdFrom,
+};
+use daphne::{
+    messages::{AggregationJobId, AggregationJobInitReq, PartialBatchSelector, TaskId},
+    DapVersion,
+};
+use serde::{Deserialize, Serialize};
+use std::{ops::Deref, slice};
+
+super::define_do_binding! {
+    const BINDING = "AGGREGATION_JOB_SET";
+
+    enum Command {
+        NewJob = "/new-job",
+        ListJobIds = "/job-ids",
+    }
+
+    fn name(
+        (version, task_id, batch_selector):
+        (DapVersion, &'n TaskId, &'n PartialBatchSelector)
+    ) -> ObjectIdFrom {
+        ObjectIdFrom::Name(format!("{version}/task/{task_id}/selector/{batch_selector}"))
```

Comment:
Should the batch selector be part of the name? This is one of the parameters of the aggreagtion job that should be illegal to change. If the Leader changes it, then we won't notice, right?

Reply:
You're right! Removed it